pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -596,3 +605,15 @@ func setChaosResourceImage() {
 		engine.Instance.Spec.Components.Runner.Image = ChaosRunnerImage
 	}
 }
+
+func getChaosType() error {
+
+	if engine.Instance.Spec.ChaosType == "" {
+		engine.Instance.Spec.ChaosType = chaosTypes.DefaultChaosType
```

Comment:
@chandankumar4 , If we will use a chaosType variable here to store engine.Instance.Spec.ChaosType then we have to update that variable in this if condition also. so It will increase overhead.
something like  : 
`
 if (chaosType == "" ){
engine.Instance.Spec.ChaosType = chaosTypes.DefaultChaosType
chaosType = chaosTypes.DefaultChaosType
}
```

Reply:
okay!!